source/core/NumericString.cpp

```diff
@@ -1953,6 +1954,97 @@ VIREO_FUNCTION_SIGNATUREV(StringScan, StringScanParamBlock)
     elementType->CopyData(input.Begin(), _Param(StringRemaining)->Begin(), input.Length());
     return _NextInstruction();
 }
+
+struct ByteArrayToStringParamBlock : InstructionCore
+{
+    _ParamDef(StringRef, StringOut);
+    _ParamDef(TypedArrayCoreRef, ByteArrayIn);
+    _ParamDef(Int32, StringEncodingIn);  // 0 = UTF-8 (only supported)
+    _ParamDef(ErrorCluster, ErrorInOut);
+    NEXT_INSTRUCTION_METHOD()
+};
+
+static bool IsSupportedEncodingAndUpdateErrorAndMakeOutputArrayEmpty(ErrorCluster* errorCluster, Int32 stringEncoding, TypedArrayCoreRef outputArray) {
+    if (stringEncoding != 0) {
+        if (errorCluster != nullptr) {
```

Comment:
I remember seeing somewhere in the code that we should use null instead of of nullptr.